Neos.ContentRepository/Tests/Behavior/Features/Bootstrap/CurrentSubgraphTrait.php

```diff
@@ -125,6 +125,7 @@ protected function getCurrentSubgraphs(): ContentSubgraphs
     {
         $currentSubgraphs = [];
         foreach ($this->getActiveContentGraphs() as $adapterName => $contentGraph) {
+            $contentGraph->disableCache();
```

Comment:
uh, why is this needed? I am not sure this is correct actually, but it might mask actual errors during the tests.

Reply:
Now that you mention it, there seem to be caching issues that should be resolved instead. Will have a look at it

Reply:
Done